brayns/common/TransferFunction.cpp

```diff
@@ -83,4 +81,14 @@ Vector4f TransferFunction::getColorForValue(const float v) const
 
     return glm::mix(color1, color2, remainder);
 }
+
+ModifiedFlag &TransferFunction::getModifiedFlag() noexcept
+{
+    return _flag;
+}
+
+const ModifiedFlag &TransferFunction::getModifiedFlag() const noexcept
+{
+    return _flag;
+}
```

Comment:
The tf modified state cannot be handled at the upper level like the transform ?

So we can remove the modified flag from it and the transfer function only contains a minimal set of data.

And it could even be a struct with public field like the transform...